backend/routes/indexer.go

```diff
@@ -384,3 +387,18 @@ func processNewDayEvent(event IndexerEvent, w http.ResponseWriter) {
 		}
 	}
 }
+
+func sendNFTWebSocketMessage(tokenID int64) {
+    message := map[string]int64{"token_id": tokenID} // Message to be sent
```

Comment:
Move this line to process nft minted event and add the "messageType": "nftMinted" field. Then use this `sendNFTWebSocketMessage` as a generic `sendWebSocketMessage` function